doc/reference/trd-unrestricted-allow-syscalls.md

```diff
@@ -0,0 +1,150 @@
+Unrestricted Allow System Call
```

Comment:
I disagree with Brad on this: it should not be called 'Unrestricted' because of course there are restrictions. I would call it "Userspace Readable Allow System Call". 

Let's schedule a voice call to try to figure this out?

Reply:
I support "Userspace Readable".

Also I didn't intend to try to expand the scope, I just didn't want to be the person who says "change the name!" but then doesn't have any ideas.